### DISCUSSION TITLE
result of tpv5

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#573] [Q&A] [answered]
I run SeisSol with  ./SeisSol_Release_dhsw_6_elastic parameters.par , I can get these files,
![image](https://user-images.githubusercontent.com/104721358/172552885-5b89163f-4936-4ffe-ad9b-4a78000e8380.png)
I open the `tpv5-fault.xdmf ` file, I just get the result.
![image](https://user-images.githubusercontent.com/104721358/172554560-7a577cb5-6104-4e3d-b8a4-671ff79529e5.png)

how can I get this picture,
![image](https://user-images.githubusercontent.com/104721358/172553986-bb6038cf-e349-4da1-ac9a-3e4f88b6e996.png)

**Comment by Thomas-Ulrich** [comment]
Try
![image](https://user-images.githubusercontent.com/13536910/172561246-34ed8531-d79d-47d6-88b0-7720c669046a.png)

**Reply by smallriver666** [reply]
> Try ![image](https://user-images.githubusercontent.com/13536910/172561246-34ed8531-d79d-47d6-88b0-7720c669046a.png)
I click `last frame`, but I get the same result.
Is this related to the paraview version, or is it possible that it is because I use `paraview --mesa` to open paraview.
![image](https://user-images.githubusercontent.com/104721358/172562605-898e3676-3397-4475-be64-2c10e80dd4e4.png)

**Reply by smallriver666** [reply]
> Try ![image](https://user-images.githubusercontent.com/13536910/172561246-34ed8531-d79d-47d6-88b0-7720c669046a.png)

and I encouter this problem if I open paraview with `paraview`
![image](https://user-images.githubusercontent.com/104721358/172564240-cf42dbd2-c3fb-4fa0-b3c8-0207f0e9ab7a.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
change partition to SRs
![image](https://user-images.githubusercontent.com/13536910/172564628-5d054279-c33c-49c7-b686-fdc83d9dae17.png)
https://seissol.readthedocs.io/en/latest/fault-output.html#outputmask

**Reply by smallriver666** [reply]
> change partition to SRs ![image](https://user-images.githubusercontent.com/13536910/172564628-5d054279-c33c-49c7-b686-fdc83d9dae17.png) https://seissol.readthedocs.io/en/latest/fault-output.html#outputmask

I get it, I extremly express my heartfelt thanks .

**Reply by Thomas-Ulrich** [reply]
Thx, please use `mark as answer` when the discussion is resolved.

**Reply by smallriver666** [reply]
> Thx, please use `mark as answer` when the discussion is resolved.

okok,get it.
